app/views/users/index.html.erb

```diff
@@ -7,6 +7,11 @@
 		<li>
 			<%= gravatar_for user %> 
 			<%= link_to user.name, user %> 
+      <% if current_user.admin? && !current_user?(user) %>
```

Comment:
Since you've got a `current_user` helper already, what about just doing `&& current_user !== user`